supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -29,51 +29,44 @@ object SharedFileSystem {
     }
   }
 
-  type PathsPair = (Path, Path)
-  type DuplicationStrategy = (Path, Path) => Try[Unit]
+  type FilesPair = (File, File)
+  type DuplicationStrategy = (File, File) => Try[Unit]
```

Comment:
I would really like to ADT these :)

At the very least it should be `DuplicationStrategy = FilesPair => Try[Unit]`